scripts/transfer_issues.py

```diff
@@ -0,0 +1,461 @@
+            processed_issues.add(issue["number"])
+
+            # Check for subissues if enabled
+            if include_subissues and issue.get("body"):
+                subissue_refs = extract_subissue_references(issue["body"], SOURCE_REPO)
+                if subissue_refs:
+                    print(f"  Issue #{issue['number']} references subissues: {subissue_refs}")
+                    for sub_num in subissue_refs:
+                        # Track parent-child relationship
+                        if sub_num not in parent_child_map:
+                            parent_child_map[sub_num] = []
+                        parent_child_map[sub_num].append(issue["number"])
+
+                        if sub_num not in processed_issues:
+                            subissue = fetch_issue_by_number(SOURCE_REPO, sub_num)
+                            if subissue and subissue.get("state") == "OPEN":
+                                issues_to_process.append(subissue)
+                                processed_issues.add(sub_num)
+                                print(f"    Added subissue #{sub_num} to transfer queue")
+
+    # Sort issues so subissues are processed before their parents
+    # This ensures we have the new issue numbers before updating parent references
+    def get_issue_depth(issue_num, visited=None):
+        if visited is None:
+            visited = set()
+        if issue_num in visited:
+            return 0
+        visited.add(issue_num)
+        if issue_num not in parent_child_map:
+            return 0
+        return 1 + max(get_issue_depth(parent, visited) for parent in parent_child_map[issue_num])
+
+    issues_to_process.sort(key=lambda x: get_issue_depth(x["number"]))
+
+    print(f"\nTotal issues to transfer (including subissues): {len(issues_to_process)}")
+    print()
+
+    if not issues_to_process:
+        print("No issues to transfer")
+        return
+
+    transferred = 0
+    failed = 0
+    issue_mapping = {}  # Map old issue numbers to new URLs
+    new_parent_child_map = {}  # Map new issue numbers to their parent issues
+
+    for issue in issues_to_process:
+        number = issue["number"]
+        title = issue["title"]
+        body = issue.get("body", "")
+
+        # Extract labels, assignees, milestone
+        labels = [label["name"] for label in issue.get("labels", [])]
+        assignees = [assignee["login"] for assignee in issue.get("assignees", [])]
+        milestone_data = issue.get("milestone")
+        milestone = milestone_data.get("title") if milestone_data else None
+
+        print("-" * 40)
+        print(f"Processing Issue #{number}: {title}")
+
+        # Translate labels
+        if labels:
+            translated_labels = translate_labels(labels)
+            if translated_labels != labels:
+                print(
+                    f"  Translating labels: {', '.join(labels)} -> {', '.join(translated_labels)}"
+                )
+                labels = translated_labels
+
```

Comment:
From the description:
> This will transfer labels if they exist on both repos and can translate one label to another. 

I think we should add these labels ahead of time (also to help us to agree on our label set), this is:
- https://github.com/ethereum/execution-spec-tests/issues/2200

My suggestion would be to error out (or verbosely skip) if the label doesn't exist in the target repo and abort the transfer. If successfully transferred issues get closed, it should be trivial to run the script again without any side effects.